mocks/apiServer.js

```diff
@@ -37,32 +39,45 @@ server.use((req, res, next) => {
 });
 
 server.get('/resources/time-entries', function (req, res) {
```

Comment:
All endpoints should be configurable in one place

Reply:
Agreed to discuss this in next tech huddle and open a separate ticket for it